lib/course_planner_web/templates/task/form.html.eex

```diff
@@ -12,8 +12,8 @@
               </div>
             <% end %>
             <%= CoursePlannerWeb.SharedView.form_text f, :name, required: true %>
-            <%= CoursePlannerWeb.SharedView.form_datetime f, :start_time, required: true, value: %{hour: 8, min: 0} %>
```

Comment:
why removing of the value ?

Reply:
The view's function puts the current time as a default, I thought it was supposed to be like that before but it wasn't exactly.

Reply:
in case you have checked that it puts it as default then ignore this comment